django-testbed/frontend/src/views/UserListView.js

```diff
@@ -2,20 +2,23 @@ import React, { useState } from 'react';
 import { Button, Modal } from 'antd';
 import qs from 'qs';
 import { FieldFormatter } from '@prestojs/ui';
-import { useViewModelCache } from '@prestojs/viewmodel';
+import { useViewModelCache, useViewModelChangeEffect } from '@prestojs/viewmodel';
```

Comment:
I thought we discussed changing this to `useViewModelChange` or `useViewModelState`